examples/FeatureSelection.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
It would be nice to render this in the documentation directly. Do you know if it is possible?

Reply:
Hm, I'd have to look into it